RecFCCeeCalorimeter/src/components/CalibrateBenchmarkMethod.h

```diff
@@ -0,0 +1,105 @@
+#ifndef RECCALORIMETER_CALIBRATEBENCHMARKMETHOD_H
+#define RECCALORIMETER_CALIBRATEBENCHMARKMETHOD_H
+
+// GAUDI
+#include "GaudiAlg/GaudiAlgorithm.h"
+
+#include "TObject.h"
+#include "TH1F.h"
+#include "TH2F.h"
+#include "TMath.h"
+#include "Math/Minimizer.h"
+#include "Math/Factory.h"
+#include "Math/Functor.h"
+#include "TRandom2.h"
+#include "TError.h"
+#include <iostream>
+#include "Math/IFunction.h"
+
+// Key4HEP
+#include "k4FWCore/DataHandle.h"
+class IGeoSvc;
+
+// EDM4HEP
+namespace edm4hep {
+  class CalorimeterHitCollection;
+  class MCParticleCollection;
+}
+
+namespace DD4hep {
+namespace DDSegmentation {
+class Segmentation;
+}
+}
+
+class TH1F;
+class ITHistSvc;
+
+/** @class CalibrateBenchmarkMethod CalibrateBenchmarkMethod.h
+ *
+ *  Sums energy deposited in every calorimeter layer separately, sums also energy deposited in the dead material of the
+ *  calorimeter (cryostat).
```

Comment:
Might be useful to state more explicitly here what these sums are useful for and what tools would be ensuring the next step (when they will be available)

Reply:
Sorry, I was too quick with this update and I forgot to update the description in the header file - it's fixed now. Thanks for spotting it